frontend/now.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "almond-frontend",
+  "public": true,
+  "version": 2,
+  "builds": [
+    {
+      "src": "next.config.js",
+      "use": "@now/next"
+    }
+  ],
+  "alias": "nextjs-redux-starter.now.sh",
+  "env": {
+    "GITHUB_API_ENDPOINT": "https://api.github.com"
+  }
+}
```

Comment:
I understand you have this for testing / demo, but please do not commit it. We'll push this and then demo in almond-dev.stanford.edu as usual.

Reply:
Will remove